swift-sdk/Internal/OfflineRequestProcessor.swift

```diff
@@ -259,6 +259,20 @@ struct OfflineRequestProcessor: RequestProcessorProtocol {
                                    identifier: #function)
     }
     
```

Comment:
Why is dismiss left out from the InternalIterableAPI?

Reply:
I'm not sure if it's necessary since the methods are available through the embedded messaging manager, but I added them for now anyways.